### PR TITLE
logging updates

### LINKED ISSUE #384: update code for M1 release
- update version to 0.1.0
- package updates
- TODO
- tag repo with version

### PR DESCRIPTION
**Pull request by bartr**
# Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
- update logging for efficiency
- reduce creation / garbage collection of log objects

## Validation

- [x] Unit tests updated and ran successfully
- [x] Update documentation or issue referenced above

## Issues Closed or Referenced

- References #376 
- Closes #384

### CODE REVIEW

File: src/Ngsa.DataService/Controllers/ResultHandler.cs [hunk 1]
```diff
@@ -49,22 +45,19 @@ public static async Task<IActionResult> Handle<T>(Task<T> task, NgsaLog logger)
...
+                logger.LogError(new EventId((int)ce.StatusCode, "CosmosException"), nameof(Handle), $"CosmosActivityId: {ce.ActivityId}", ex: ce);
 
                 return CreateResult(logger.ErrorMessage, ce.StatusCode);
             }
             catch (Exception ex)
             {
                 // log and return exception
-                logger.EventId = new EventId((int)HttpStatusCode.InternalServerError, "Exception");
-                logger.LogError($"Exception: {ex.Message}", ex);
+                logger.LogError(new EventId((int)HttpStatusCode.InternalServerError, "Exception"), nameof(Handle), "Exception: {ex.Message}", ex: ex);
```
Comment (jofultz) [inline src/Ngsa.DataService/Controllers/ResultHandler.cs:60]: Do the nameof(Handle) print something like "System.Activities.Handle" as part of the error log?  If so, we use the name of whatever T is for the log?

File: src/Ngsa.Middleware/NgsaLog.cs [hunk 2]
```diff
@@ -12,142 +12,161 @@ namespace Ngsa.Middleware
...
+                    d.Add("ExceptionMessage", ex.Message);
+                }
 
-            Console.ForegroundColor = ConsoleColor.Yellow;
-            Console.WriteLine(JsonSerializer.Serialize(d, Options));
-            Console.ResetColor();
+                // display the error
+                Console.ForegroundColor = ConsoleColor.Red;
+                Console.Error.WriteLine(JsonSerializer.Serialize(d, Options));
+                Console.ResetColor();
```
Comment (jofultz) [inline src/Ngsa.Middleware/NgsaLog.cs:103]: feels like we should move the above 3 lines to a common function and have each login function call it with something like:
`WriteLogEntry(ConsoleColor.Red, d)`

Thoughts?

**Review by jofultz — APPROVED** [review]
A couple of notes.  Mainly concerned about the nameof(Handle) item, but if it prints what is desired then I'm fine with it.

File: src/Ngsa.Middleware/NgsaLog.cs (hunk 2 above)
Comment (bartr) [reply src/Ngsa.Middleware/NgsaLog.cs]: changed per code review

**Pull request merged**